src/User_Command.php

```diff
@@ -722,27 +734,35 @@ public function add_role( $args, $assoc_args ) {
 	 * <user>
 	 * : User ID, user email, or user login.
 	 *
-	 * [<role>]
-	 * : A specific role to remove.
+	 * [<role>...]
+	 * : Remove the specified role(s) from the user.
 	 *
 	 * ## EXAMPLES
 	 *
 	 *     $ wp user remove-role 12 author
 	 *     Success: Removed 'author' role for johndoe (12).
 	 *
+	 *     $ wp user remove-role 12 author editor
+	 *     Success: Removed 'author' role for johndoe (12).
+	 *     Success: Removed 'editor' role for johndoe (12).
+	 *
 	 * @subcommand remove-role
 	 */
 	public function remove_role( $args, $assoc_args ) {
 		$user = $this->fetcher->get_check( $args[0] );
 
 		if ( isset( $args[1] ) ) {
-			$role = $args[1];
+			$roles = $args;
+			array_shift( $roles );
 
-			self::validate_role( $role );
-
-			$user->remove_role( $role );
+			foreach ( $roles as $role ) {
+				self::validate_role( $role );
+			}
 
-			WP_CLI::success( "Removed '{$role}' role for {$user->user_login} ({$user->ID})." );
+			foreach ( $roles as $role ) {
+				$user->remove_role( $role );
+				WP_CLI::success( "Removed '{$role}' role for {$user->user_login} ({$user->ID})." );
```

Comment:
Rather than render `Success:` multiple times, can we consolidate into one `Success:`?